website/docs/guides/migration/versions/01-upgrading-to-v1.6.md

```diff
@@ -22,22 +22,40 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ### Behavior changes
 
-**Coming soon**
-
+- dbt Core v1.6 does not support Python 3.7, which reached End Of Life on June 23. Support Python versions are 3.8, 3.9, 3.10, and 3.11.
+- As part of the Semantic layer re-launch (in beta), the spec for `metrics` has changed significantly. Migration guide coming soon: https://github.com/dbt-labs/docs.getdbt.com/pull/3705
+- Manifest schema version is now v10, reflecting [TODO] changes
 
```

Comment:
Discussion for adapter maintainers: https://github.com/dbt-labs/dbt-core/discussions/7958